driver.go

```diff
@@ -78,17 +84,46 @@ func openDriverConn(ctx context.Context, d *Driver, name string) (driver.Conn, e
 	if err != nil {
 		return nil, err
 	}
-	return &conn{client: client}, nil
+
+	adminClient, err := createAdminClient(ctx)
+	if err != nil {
+		return nil, err
+	}
+	return &conn{client: client, adminClient: adminClient, name: name}, nil
+}
+
+func createAdminClient(ctx context.Context) (adminClient *adminapi.DatabaseAdminClient, err error) {
+
+	// Admin client will connect to emulator if SPANNER_EMULATOR_HOST
+	// is set in the environment.
+	if spannerHost, ok := os.LookupEnv("SPANNER_EMULATOR_HOST"); ok {
+		adminClient, err = adminapi.NewDatabaseAdminClient(
+			ctx,
+			option.WithoutAuthentication(),
+			option.WithEndpoint(spannerHost),
+			option.WithGRPCDialOption(grpc.WithInsecure()))
+		if err != nil {
+			return
```

Comment:
These error checks are currently superfluous due to the later return statement. We could either remove them and return the client and error as is from each call; or we could set the `adminClient` to `nil` if there is an error.

Technically speaking, either option is fine, but I think setting the client to `nil` is better just in case someone doesn't check the error.

Reply:
set admin client to nil if err, returns at end